WcaOnRails/spec/controllers/competitions_controller_spec.rb

```diff
@@ -213,6 +213,12 @@
         sign_in organizer
       end
 
+      it 'cannot confirm competition' do
+        patch :update, id: competition, competition: { name: competition.name }, commit: "Confirm"
```

Comment:
That's a good point, this test is a little scary. However, a new route seems problematic, as we only have 1 form here. Are you envisioning writing some javascript client side to dispatch the form submission to the correct route?